templates_test.go

```diff
@@ -377,3 +380,55 @@ func TestTemplateModeRemote(t *testing.T) {
 }
 
 // TODO: test typed errors for custom and remote (embedded checked)
+
+func TestTemplateFuncYAML(t *testing.T) {
```

Comment:
It helps quite a bit to have all tests include a statement of purpose as their doc string, and a function name that is as similar as possible. 
Perhaps something like the below?
```suggestion
// TestTemplateManifestBuildEnvs ensures that BuildEnvs specified in a
// template's manifest are included in the final Function.
func TestTemplateManifestBuildEnvs(t *testing.T) {a
```

Reply:
👍